### PR TITLE
[FEAT] Banana

### PR DESCRIPTION
**Pull request by c-py**
# Description

Adds Bananas in game

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: metadata/metadata.ts [hunk 1]
```diff
@@ -214,6 +214,18 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
...
   },
+  "Banana Plant": {
+    name: "Banana Plant",
+    description:
+      "A plant used to grow bananas.\n\nYou can buy banana plants in game at the market.",
+    decimals: 18,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image_url: "../public/erc1155/images/120.png",
+    attributes: [
+      { trait_type: "Purpose", value: "Seed" },
```
Comment (brian-man) [inline metadata/metadata.ts:225]: Does this `Purpose` imply Kuebiko should impact the price?

Comment (c-py) [reply metadata/metadata.ts:225]: @brian-man Yeah right now Kuebiko will impact the price.

File: public/erc1155/images/120.png
Comment (elias7905) [inline public/erc1155/images/120.png:1]: Hi @c-py I'm wondering if this is just a placeholder for the banana plant or is this the sprite that you've decided to use for it?
I'm aware that bananas don't usually grow from seeds, like u mentioned on stream, but maybe we could use a sprite of a more grown out banana plant as its image?

**Review by spencerdezartsmith — APPROVED** [review]
🍌

File: public/erc1155/images/120.png
Comment (c-py) [reply public/erc1155/images/120.png:1]: @elias7905 Yeah for now this will be the image - I tried creating one - but my art-fu wasn't strong enough

**Pull request merged**